functorch/csrc/BatchRulesHelper.cpp

```diff
@@ -83,6 +83,24 @@ Tensor maybePadToLogicalRank(const Tensor& tensor, optional<int64_t> has_bdim, i
   return tensor.view(new_sizes);
 }
 
+void check_randomness(RandomnessType randomness, bool any_tensor_batched) {
+  TORCH_CHECK(
+    randomness != RandomnessType::Error,
+    "vmap: called random operation while in randomness error mode. Please either use the "
```

Comment:
nit: would be nice if the name of the random operation was included in the error message